person.rb

```diff
@@ -5,7 +5,7 @@ def correct_name
 end
 
 class Person < Nameable
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
```

Comment:
- [ ] You did well to add a rental as a getter/setter, but we need to associate it using a method. Kindly add a method that will link a person and a rental, achieving the `has_many` rentals associations.

```Ruby
def add_rental(book, date)
    Rental.new(date, book, self)
end
```